specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-08-01-beta/analysisservices.json

```diff
@@ -379,6 +379,44 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{serverName}/listGatewayStatus": {
+      "post": {
+        "description": "Return the gateway status of the specified Analysis Services server instance.",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_ListGatewayStatus",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "serverName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the Analysis Services server. It must be at least 3 characters in length, and no more than 63."
```

Comment:
Please remove ` It must be at least 3 characters in length, and no more than 63.`, don't restate constraints expressed in Swagger by other means.